Code/Editor/Lib/Tests/test_ModularViewportCameraController.cpp

```diff
@@ -427,4 +446,58 @@ namespace UnitTest
         // Clean-up
         HaltCollaborators();
     }
+
+    TEST_F(ModularViewportCameraControllerFixture, CameraModifersAreDetectedWhenViewportIsNotInFocus)
+    {
+        SandboxEditor::SetCameraCaptureCursorForLook(false);
+
+        const float deltaTime = 1.0f / 60.0f;
```

Comment:
This can be marked `constexpr`

Reply:
Sure thing I can add that 👍

Reply:
FYI this is now updated to be `constexpr` and I also extracted this to a file-level constant as it was the same in all tests